ToolBox/Visualizer/Visualizer.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using Python.Runtime;
+using QuantConnect.Python;
+
+namespace QuantConnect.ToolBox.Visualizer
+{
+    public class Visualizer : LeanDataReader
+    {
+        private readonly PandasConverter _converter = new PandasConverter();
+
+        /// <inheritdoc />
+        public Visualizer(string filepath)
+            : base(filepath)
+        { }
+
+        public PyObject ParseDataFrame() => _converter.GetDataFrame(Parse());
```

Comment:
it seems like this entire class exists just to call `new PandasConverted().GetDataFrame( <output from leandatareader.parse )` -- is this class needed -- or do we just need a function to create a data frame from an ienumerable of base data?

Also, it doesn't seem this class does any 'visualizing' -- so perhaps the name is just throwing me off

Reply:
Yes it has some smell, it can be just a method in the `LeanDataReader` (Lazy class / freeloader).